internal/unionstore/art/art.go

```diff
@@ -115,6 +125,7 @@ func (t *ART) Set(key artKey, value []byte, ops ...kv.FlagsOp) error {
 		}
 	}
 
+	t.WriteSeqNo++
```

Comment:
Though there is no such usage in TiDB.
Will you consider add `SnapshotSeqNo` here if `len(t.stages) == 0`?

Reply:
It won't hurt. Let's do it